source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -705,9 +614,6 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering
             // If the image hasn't loaded yet
             if (imageFiresOpenEvent)
             {
-                // Collapse the Ellipse while the image loads, so that resizing is not noticeable
```

Comment:
What was this used for before?

Reply:
It's side effects of using co-routine. we don't need to make it invisible.